### DISCUSSION TITLE
Go Mailer

### DISCUSSION THREAD

**Discussion by DisgruntledTech93** [sysadminsmedia/homebox#176] [General]
I'm looking to set up Go Mailer to send out emails about specific actions. Does anyone know of a working walkthrough for setting this up?

**Comment by tankerkiller125** [comment]
Are you referring to Go Mailer the service? Or gomailer the library used for sending SMTP emails?

**Comment by DisgruntledTech93** [comment]
Not exactly to sure, I know I was looking at some of the config and the
system can send emails(notifications), I just don’t know the process for
getting that set up

**Reply by tankerkiller125** [reply]
We support SMTP built in via environment variables:

```
HBOX_MAILER_HOST
HBOX_MAILER_PORT
HBOX_MAILER_USERNAME
HBOX_MAILER_PASSWORD
HBOX_MAILER_FROM
```

**Reply by codersaur** [reply]
I've set these env vars in my docker compose, but how can I test its actually working? There doesn't seem to be any option to test SMTP settings from the Web UI?

If I set up a Notifier in the Web UI (using the shoutrrr syntax) this seems to require all the details again. Do shoutrrrr notifiers mean the env vars are deprecated? Should I be using one or the other for SMTP notifications?

**Comment by DisgruntledTech93** [comment]
Oh so no extra setup just add my configuration, thank you
